### PR TITLE
added functions for writing multiple lines or to a specific line (and documentation)

### PR DESCRIPTION
**Pull request by TomW1605**

### CODE REVIEW

File: RPLCD/lcd.py [hunk 1]
```diff
@@ -67,10 +67,6 @@
 # Flags for display/cursor shift
 LCD_DISPLAYMOVE = 0x08
 LCD_CURSORMOVE = 0x00
-
```
Comment (TomW1605) [inline RPLCD/lcd.py]: this section is a duplicate of the bit above and therefore not nesasery

File: RPLCD/lcd.py [hunk 2]
```diff
@@ -67,10 +67,6 @@
...
-# Flags for display/cursor shift
-LCD_DISPLAYMOVE = 0x08
-LCD_CURSORMOVE = 0x00
```
Comment (TomW1605) [inline RPLCD/lcd.py]: this section is a duplicate of the bit above and therefore not nesasery

Comment (dbrgn) [inline RPLCD/lcd.py]: Thanks for noticing.

**Comment by dbrgn** [comment]
Hi. Thanks for the contribution. If I understand it correctly, this

```python
lcd.write_line('hello', 2)
```

...is the same as this?

```python
lcd.cursor_pos = (2, 0)
lcd.auto_linebreaks = False
lcd.write_string('hello')
```

**Comment by TomW1605** [comment]
Basically yes

**Comment by dbrgn** [comment]
Ok. In that case thanks again for contributing, but I probably won't merge this. I can see that it can be useful as a shortcut in certain very specific use cases, but it's a redundant feature that at the same time increases the complexity of the codebase and the documentation (multiple ways of writing strings to the display with slightly different behavior).

If you disagree, please reopen. And if you want to discuss other potential API changes, feel free to open an issue so we can discuss the design before writing the implementation :)

**Comment by TomW1605** [comment]
thanks for considering it i mostly wrote it because i started using this after the library i was using didn't support custom caricatures and i wanted to use the same systems in my code with this library as i had with the old one. i just thought other uses might like the simple one liner to write to all lines on the lcd. if u dont mind i will continue to use this code in my custom library.

**Comment by dbrgn** [comment]
Of course, feel free to reuse and distribute modified code. I think it may suit your use case well :) But as maintainer you need to be careful not to add too many features that you need to maintain later on... :)

**Comment by TomW1605** [comment]
I completely agree as I am having problems with having made that mistake in my most recent project

________________________________

This email and any files transmitted with it are confidential and intended solely for the use of the individual or entity to whom they are addressed. If you have or believe you may have received this email in error please notify the sender immediately and delete this email from your system. If you are not the intended recipient you are notified that disclosing, copying, distributing or taking any action in reliance on the contents of this information is strictly prohibited. The views or opinions presented in this email are solely those of the author or sender and do not necessarily represent those of Scotch College. It is the responsibility of the recipient to check this email and any attachments for the presence of viruses. Scotch College accepts no liability for any damage caused by any virus transmitted by this email.

**Pull request closed without merging**